### PR TITLE
ci: harden chrono test, mark another macos 4.9 dev failure

### PR DESCRIPTION
**Pull request by henryiii**
This should help with a little of the flakiness seen with the timing test

### CODE REVIEW

**Review by YannickJadoul — APPROVED** [review]

File: tests/test_chrono.py [hunk 1]
```diff
@@ -9,21 +9,23 @@
...
     diff = abs(date1 - date2)
 
     # There should never be a days difference
     assert diff.days == 0
 
     # Since datetime.datetime.today() calls time.time(), and on some platforms
-    # that has 1 second accuracy, we should always be less than 2 seconds.
-    assert diff.seconds < 2
+    # that has 1 second accuracy, we compare this way
+    assert int(diff.seconds) <= int(diff_python.seconds)
```
Comment (YannickJadoul) [inline tests/test_chrono.py:28]: Why `int` here, though?

Comment (henryiii) [reply tests/test_chrono.py:28]: I didn't check to make sure it was forced to be an int already. Is it?

Comment (henryiii) [reply tests/test_chrono.py:28]: I think it is. The docs say it is `in range(60)`.

Comment (henryiii) [inline tests/test_chrono.py:28]:
```suggestion
    assert diff.seconds <= diff_python.seconds
```

Comment (YannickJadoul) [reply tests/test_chrono.py]: OK. The idea was mainly that even if it was something else, `<=` still ought to work? :-) But good!

**Pull request merged**